google-api-client/src/test/java/com/google/api/client/googleapis/apache/v2/GoogleApacheHttpTransportTest.java

```diff
@@ -0,0 +1,71 @@
+package com.google.api.client.googleapis.apache.v2;
+
+import java.io.InputStream;
+import java.security.KeyStore;
+
+import com.google.api.client.googleapis.util.Utils;
+import com.google.api.client.http.apache.v2.ApacheHttpTransport;
+import com.google.api.client.util.SecurityUtils;
+
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import junit.framework.TestCase;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({Utils.class, GoogleApacheHttpTransport.class})
+@PowerMockIgnore({"jdk.internal.reflect.*", "javax.net.ssl.*"})
+public class GoogleApacheHttpTransportTest extends TestCase {
+  public KeyStore createTestMtlsKeyStore() throws Exception {
+    InputStream certAndKey = getClass()
+      .getClassLoader()
+      .getResourceAsStream("com/google/api/client/googleapis/util/mtlsCertAndKey.pem");
+    return SecurityUtils.createMtlsKeyStore(certAndKey);
+  }
+
+  // If client certificate shouldn't be used, then neither the provided mtlsKeyStore
+  // nor the default mtls key store should be used.
+  public void testNotUseCertificate() throws Exception {
+    KeyStore mtlsKeyStore = createTestMtlsKeyStore();
+    PowerMockito.spy(Utils.class);
+    PowerMockito.when(Utils.useMtlsClientCertificate()).thenReturn(false);
+    PowerMockito.when(Utils.loadDefaultMtlsKeyStore()).thenReturn(mtlsKeyStore);
```

Comment:
I think needing to mock static methods is a bit of a code-smell - we can wrap these helpers into a provider class which we can mock/inject for testing.